src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -338,3 +338,43 @@ void AppHost::_ToggleFullscreen(const winrt::Windows::Foundation::IInspectable&,
 {
     _window->ToggleFullscreen();
 }
+
+// Method Description:
+// - Called when the IslandWindow has received a WM_MOUSEWHEEL message. This can
+//   happen on some laptops, where their trackpads won't scroll inactive windows
+//   _ever_.
+// - We're going to take that message and manually plumb it through to our
+//   TermControl's, or anything else that implements IMouseWheelListener.
+// - See GH#979 for more details.
+// Arguments:
+// - coord: The Window-relative, logical coordinates location of the mouse during this event.
+// - delta: the wheel delta that triggered this event.
+// Return Value:
+// - <none>
+void AppHost::_WindowMouseWheeled(const til::point coord, const int32_t delta)
+{
+    if (_logic)
+    {
+        // Find all the elements that are underneath the mouse
+        auto elems = winrt::Windows::UI::Xaml::Media::VisualTreeHelper::FindElementsInHostCoordinates(coord, _logic.GetRoot());
+        for (const auto& e : elems)
+        {
+            // If that element has implemented IMouseWheelListener, call OnMouseWheel on that element.
+            if (auto control{ e.try_as<winrt::Microsoft::Terminal::TerminalControl::IMouseWheelListener>() })
+            {
+                // Translate the event to the coordinate space of the control
+                // we're attempting to dispatch it to
+                const auto transform = e.TransformToVisual(nullptr);
+                const auto controlOrigin = transform.TransformPoint(til::point{ 0, 0 });
+                const til::point offsetPoint{ ::base::ClampSub(coord.x(), controlOrigin.X), ::base::ClampSub(coord.y(), controlOrigin.Y) };

```

Comment:
Add a clamped sub method to til::point instead of doing it on the outside? I feel like this line should read

```
const auto offsetPoint = coord.ClampSub(controlOrigin);
```

Reply:
Okay so the real reason I was doing this _wasn't_ for clamped math but more because I couldn't use the `float`s from the `Windows::Foundation::Point`. Now that I _can_, I'll try/catch this appropriately.